go/README.md

```diff
@@ -0,0 +1,23 @@
+# Go Kinesis Deaggregation Modules
+
+The Kinesis Deaggregation Modules for Go provide the ability to do in-memory deaggregation of standard Kinesis user records using the [Kinesis Aggregated Record Format](https://github.com/awslabs/amazon-kinesis-producer/blob/master/aggregation-format.md) to allow for more efficient transmission of records.
+
+There are 2 versions based upon the AWS SDK version you are using:
+
+| SDK | Project |
+| --- | ------- |
+|Version 0 | [v0](.) |
```

Comment:
Is the first version referred to as v0 or v1? I'd assume v1

Reply:
when importing it it is `github.com/awslabs/kinesis-aggregation/go v0.0.0-20210630091500-54e17340d32f` so I left it as v0

Reply:
Gotcha. so versioning it based on THIS repo vs the version of AWS SDK it's being consumed from